lib/main.dart

```diff
@@ -61,24 +61,61 @@ class _CardInterfaceState extends State<CardInterface> {
   int _currentEndVerse;
   GlobalKey<AutoCompleteTextFieldState<String>> key = new GlobalKey();
 
-  SimpleAutoCompleteTextField textField;
+  AutoCompleteTextField<String> textField;
   TextEditingController bookName = new TextEditingController();
 
   //This body is for text auto-completion for book name
   _CardInterfaceState() {
-    textField = SimpleAutoCompleteTextField(
+    textField = AutoCompleteTextField(
       key: key,
-      controller: bookName,
       suggestions: suggestions,
-      textChanged: (text) => _currentBook = text,
       clearOnSubmit: false,
-      submitOnSuggestionTap: false,
+      submitOnSuggestionTap: true,
```

Comment:
What does this change actually do? Changing it to `false` doesn't seem to change the behavior of tapping to get the full name on the text field.